kimchi/src/prover.rs

```diff
@@ -477,28 +330,40 @@ where
             lookup_context.joint_lookup_table = Some(joint_lookup_table);
         }
 
-        //~ 1. Sample $\beta$ with the Fq-Sponge.
-        let beta = fq_sponge.challenge();
+        Ok(lookup_context)
+    }
 
-        //~ 1. Sample $\gamma$ with the Fq-Sponge.
-        let gamma = fq_sponge.challenge();
+    pub fn compute_aggregation_polynomial<
```

Comment:
this logic is so straight forward that I'm not sure it really makes sense to factor it out in its own function